configure.ac

```diff
@@ -131,8 +131,10 @@ AC_CHECK_FUNCS([bzero gettimeofday memmove memset mkdir strdup strrchr \
 LT_INIT
 AC_PROG_LIBTOOL
 
-# Decide if we're building on Windows early on.
 AM_CONDITIONAL([USING_WIN32], [test "x$host_os" = xmingw32])
+if test -z "${USING_WIN32_FALSE}"; then
```

Comment:
E.g. https://github.com/OpenLightingProject/ola/blob/master/common/network/InterfacePicker.cpp#L142 It kind of feels wrong using _WIN32 to decide if we're on 64 bit, but perhaps that's just what everyone does for compatability